src/main/kotlin/racingcar/core/RaceManager.kt

```diff
@@ -0,0 +1,29 @@
+package racingcar.core
+
+import racingcar.model.Car
+import racingcar.ui.ResultView
+import racingcar.ui.ResultView.printResultTitle
+
+class RaceManager(
+    private val carCount: Int,
+    private val moveCount: Int,
+    private var carList: List<Car> = initCarList(carCount),
+) {
+    fun startRacing() {
+        printResultTitle()
+
+        repeat(moveCount) {
+            carList = getUpdatedCarList(carList)
+            ResultView.printCarPositionResults(carList.map { "-".repeat(it.position) })
+        }
+    }
+
+    private fun getUpdatedCarList(carList: List<Car>): List<Car> =
+        carList.map { car ->
+            car.updateCarPosition(RaceConditionChecker.isAdvancePossible())
+        }
```

Comment:
람다 변수가 한개 뿐이라면 명시적으로 네이밍하지 않아도 동일한 값을 `it`으로 사용할 수 있어요. 😃
너무 중첩되는 상황만아니라면 저는 이 방식을 많이 활용하는 편이에요. 😁

```suggestion
        carList.map { it.updateCarPosition(RaceConditionChecker.isAdvancePossible()) }
```

Reply:
아하, 저는 웬만한 상황에선 네이밍을 해주는게 이해하기 편하지 않을까 싶어서 사용을 했는데, 이번에는 it으로 한번 사용해보겠습니다!

Reply:
이 부분은 취향을 탈 수도 있는 문제라서 꼭 `it`을 쓰자는 이야기는 아니였어요 😃
혹시나 모를실 수도 있다는 생각에 언급드릴 것이라서 이미 아시는 개념이라면 꼭 수정하지 않으셔도됩니다. 😉